pom.xml

```diff
@@ -49,7 +49,7 @@
         <!-- NOTE: when changing these versions, also update security.policy and test-framework.policy to match -->
         <!-- (we don't do any automatic property substitution on those resources, it could get confusing...     -->
         <lucene.version>5.4.0</lucene.version>
-        <lucene.snapshot.revision>1710880</lucene.snapshot.revision>
+        <lucene.snapshot.revision>1711508</lucene.snapshot.revision>
```

Comment:
i think you should remove this. i don't want there to be any confusion that these files are not to be managed. I have an open PR to remove them: #14375